src/components/Composer.vue

```diff
@@ -1684,6 +1686,11 @@ export default {
 	width: 100%;
 }
 
+:deep(.v-select.select .vs__selected) {
+	height: unset;
+	min-height: calc(var(--default-clickable-area) - var(--default-grid-baseline));
+}
+
```

Comment:
This feels hacky. Can we set a max width? The NcSelect seems to elipsize its selection when the width of the container is restrained. I'd prefer that over line wraps.